Helpers/ConfigHelper.php

```diff
@@ -223,6 +222,28 @@ public function saveBasePlansConfig(array $plans): void
         $this->cleanCache(CacheConfig::TYPE_IDENTIFIER);
     }
 
+    public function saveIsAllowedInsurance($merchant, $scope, $storeId):void
+    {
+        $isAllowedInsurance = 0;
+        if ($merchant) {
+            $isAllowedInsurance = 1;
```

Comment:
I think this line is useless

Reply:
Merchant is not strongly typed and can theoretically be null